src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView/bpp.jelly

```diff
@@ -19,24 +19,21 @@
     <script src="${rootURL}/plugin/build-pipeline-plugin/js/jquery.tooltip.min.js"></script>
     <script src="${rootURL}/plugin/build-pipeline-plugin/js/build-pipeline.js"></script>
 
-<script id="build-card-template" type="text/x-handlebars-template">
+<span id="build-card-template" type="text/x-handlebars-template" style="display: none">
```

Comment:
We can drop the `type=...` part, can't we?

Reply:
Thanks!